salt/metalk8s/defaults.yaml

```diff
@@ -51,4 +53,36 @@ kubelet:
     authorization:
       mode: AlwaysAllow
 
+ca:
+  cert:
+    name: /etc/kubernetes/pki/ca.crt
+    signing_private_key: /etc/kubernetes/pki/ca.key
+    CN: kubenetes
+    keyusage: "critical digitalSignature, keyEncipherment, keyCertSign"
+    basicConstraints: "critical CA:true"
+    days_valid: 3650
+  signing_policies:
+    api:
+      - minions: '*'
+      - signing_private_key: /etc/kubernetes/pki/ca.key
+      - signing_cert: /etc/kubernetes/pki/ca.crt
+      - keyusage: "critical digitalSignature, keyEncipherment"
+      - extendedKeyUsage: "serverAuth"
+      - days_valid: 365
```

Comment:
Is this a max value? We'll want the actual certs to have a somewhat-randomized validity.

Reply:
Oh ! Why we would like this to be random ?

Reply:
I'll put that in the requirements document. Never mind for now.